src/views/certifications/views/landing/components/signup-form-area/components/newsletter-signup-form/marketo-form-data.module.css

```diff
@@ -0,0 +1,108 @@
+.formWrapper {
+	width: 100%;
+}
+
+.form {
+	/* 
+    The following styles allow us to place the form button alongside the email field in the first
+    row. 
+  */
+
+	display: grid;
+	width: 100%;
+	row-gap: 24px;
+	margin-top: 8px;
+
+	& [data-field-id='Email'],
+	& [data-field-id='Consent_Privacy_Policy__c'] {
+		margin: 0;
+	}
+
+	& [data-field-id='Email'] label {
+		display: none;
+	}
+
+	& [data-field-id='Email'] input::placeholder {
+		color: transparent;
+	}
+
+	& button {
+		height: 50px;
+		justify-self: stretch;
+		align-self: start;
+		background-color: var(--token-color-foreground-action);
+		border-radius: var(--token-form-control-border-radius);
+	}
+
+	@media (min-width: 768px) {
+		grid-template-columns: 1fr max-content;
+		column-gap: 10px;
+		transition: opacity 0.35s;
+
+		& [data-field-id='Email'] {
+			grid-column: 1 / 2;
+			word-break: break-all;
+		}
+
+		& button {
+			grid-row: 1 / 2;
+			grid-column: 2 / 3;
+		}
+
+		& [data-field-id='Consent_Privacy_Policy__c'] {
+			grid-row: 2 / 3;
+			grid-column: 1 / -1;
+		}
+	}
+}
+
+.dark {
+	& [data-field-id='Email'] input {
+		border-radius: var(--token-form-control-border-radius);
```

Comment:
Updated what I immediately saw to match hds, there were a few things that I couldn't match to spec without the styles totally crumbling (standouts are the email input / button height and the checkbox height)